.github/workflows/packaging.yaml

```diff
@@ -2,27 +2,30 @@
 # See https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository
 # yamllint --format github .github/workflows/msi.yaml
 ---
-name: "msi"
+name: "packaging"
 
 env:  # As of 2021-08-05 images come with go 1.15 https://github.com/actions/virtual-environments/tree/main/images
   GO_VERSION: "1.17.0"
 
 on:
   push:  # We run tests on non-tagged pushes to master
-    tags: ''
-    branches: master
+    tags:
+      - ""
```

Comment:
Not sure if we really need this since this fails the validation with (https://json.schemastore.org/github-workflow.json): `String is shorter than the minimum length of 1.yaml-schema: https://json.schemastore.org/github-workflow.json`

Reply:
I recall having to add this directly in Zipkin before in order to stop this workflow from running on a release tag. I'm not sure how authoritative that schema is or not. If this is authoritative, we should note to verify the schema using it as that would assume that the current validation hint at the top of the file isn't good enough.

In any case, we'd know it was needed if the explicit empty was deleted, you push a tag to master and this ran as a result. That's an experiment that could be done in another repo as well.